tests/bumper-spec.js

```diff
@@ -1,7 +1,150 @@
 'use strict'
 
-// const expect = require('chai').expect
-// const Bumper = require('../lib/bumper')
+const expect = require('chai').expect
+const Promise = require('promise')
+const sinon = require('sinon')
+
+const utils = require('../lib/utils')
+const logger = require('../lib/logger')
+const Bumper = require('../lib/bumper')
 
 describe('Bumper', () => {
+  let bumper, sandbox
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create()
+    sandbox.stub(logger, 'log')
+    bumper = new Bumper({}, {})
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+  })
+
+  describe('.check()', () => {
+    beforeEach(() => {
+      sandbox.stub(bumper, 'getOpenPrScope').returns(Promise.resolve('minor'))
+    })
+
+    describe('when not a PR build', () => {
+      beforeEach((done) => {
+        bumper.check().then(() => {
+          done()
+        })
+      })
```

Comment:
I believe you can simply return a promise instead of using the `done` callback.
```javascript
beforeEach(function () {
  return bumper.check()
})
```